### ISSUE TITLE
Other players unable to see certain images unless they place it

### ISSUE THREAD

**Issue by TommyMP** [zDevelopers/ImageOnMap#76]
I installed the plugin and it seemed to be working fine until my friend told me that he couldn't see the images I placed on the item frames. Then if he took the map and placed it again he was able to see it but I wasn't. This isn't happening to all imagines, but to a large quantity of them. Any solutions?

**Comment by Vlammar** [comment]
We are aware of this bug, if I remember correctly if your friend go far enougth that force reload the images.

**Comment by Vlammar** [comment]
Update for this bug, we found and patched the bug. It was caused by this line https://github.com/zDevelopers/ImageOnMap/blob/indev/src/main/java/fr/moribus/imageonmap/ui/MapItemManager.java#L270 , Minecraft seems to have a buffer where it store map already rendered, that's why the client who put the map get a good render and other player near him doesn't, by going far enougth you force the server to send you the new render. By cancelling the event it seems that server doesn't send the render for the map placed.
